modules/s3/main.tf

```
@@ -493,6 +537,50 @@ resource "aws_iam_role_policy_attachment" "read_request_lambda_policy_attachment
   policy_arn = aws_iam_policy.read_request_lambda_policy.arn
 }
 
+resource "aws_iam_policy" "read_request_aggregated_lambda_policy" {
+  name        = "read_request_aggregated_lambda_policy"
+  description = "read_request_aggregated_lambda_policy"
+
+  policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": "s3:*",
+      "Effect": "Allow",
+      "Resource": [
+        "${aws_s3_bucket.ugt_requests_aggregations.arn}*",
+        "${aws_s3_bucket.ugt_requests_aggregations.arn}/*"
```

That lambda is only reading from the s3 bucket, right? If so, let's give it read only permissions.

```suggestion
      "Action": ["s3:Get*", "s3:List*"],
      "Effect": "Allow",
      "Resource": [
        "${aws_s3_bucket.ugt_requests_aggregations.arn}",
        "${aws_s3_bucket.ugt_requests_aggregations.arn}/*"
```